docs/multicluster/user-guide.md

```diff
@@ -559,6 +560,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters through the Multi-cluster Gateways.
+Pod IPs can be reached in all member clusters within a ClusterSet. If you want to enable this feature,
+the cluster's Pod CIDRs must be set in ConfigMap `antrea-mc-controller-config-**` of each member cluster like below example.
+You can edit [antrea-multicluster-member.yml](../../multicluster/build/yamls/antrea-multicluster-member.yml)
```

Comment:
Move this sentence to be before the kubectl edit command example.

Then Probably we can remove "Edit the ConfigMap by `kubectl edit` command:".
> You can edit [antrea-multicluster-member.yml](../../multicluster/build/yamls/antrea-multicluster-member.yml), or using `kubectl edit` to change the ConfigMap:

Reply:
Removed the "Edit..." and moved the "You can edit...." front of the command.